adobe/react-spectrum
# RSP-1376 Breadcrumbs isHeading support

It should be possible to use any variant of breadcrumbs as a page heading. Currently, we switch the last breadcrumb to an h1 only in the large size. We should instead change to role='heading' and aria-level={current level} if the `isHeading` prop is set to allow any variant to become a heading. Large size also shouldn't imply heading.

https://excelerate-team.slack.com/archives/CB2S6SRK4/p1573496456133100?thread_ts=1573496456133100&cid=CB2S6SRK4


Closes [RSP-1376](https://jira.corp.adobe.com/browse/RSP-1376)

## ✅ Pull Request Checklist:

- [x] Included link to corresponding issue [RSP-1376](https://jira.corp.adobe.com/browse/RSP-1376).

## 📝 Test Instructions:

1. Open http://localhost:9003/?path=/story/breadcrumbs--isheading-true
2. In browser's web inspector, verify that the current breadcrumb item link is wrapped in a `<span>` with `role="heading"` and `aria-level="1"`.
3. Open http://localhost:9003/?path=/story/breadcrumbs--default
4. In browser's web inspector, verify that the current breadcrumb item link is not wrapped in a `<span>` with `role="heading"` and `aria-level="1"`.
5. Open http://localhost:9003/?path=/story/breadcrumbs--size-l
6. In browser's web inspector, verify that the current breadcrumb item link in the multiline variant is wrapped in a `<span>` with `role="heading"` and `aria-level="1"`.

## 🧢 Your Team:

Accessibility
